test/clients/test_clients.py

```diff
@@ -1,25 +1,35 @@
+from hypothesis import given
+from hypothesis.searchstrategy.lazy import LazyStrategy
+from hypothesis.strategies import lists, text
 import pytest
-
```

Comment:
Been following having imports as separate groups (though not found a linter than can enforce it). (`test_client.py` should be separated too.)

Reply:
OK, changed.